src/imitation/policies/base.py

```diff
@@ -101,3 +121,24 @@ def get_action_policy(policy, observation, deterministic=True):
     clipped_actions = clipped_actions[0]
 
   return clipped_actions, states
+
+
+class MnistCnnPolicy(FeedForwardPolicy):
+  """
+  Policy object that implements actor critic, using a CNN (the Mnist CNN)
+  :param sess: (TensorFlow session) The current TensorFlow session
+  :param ob_space: (Gym Space) The observation space of the environment
+  :param ac_space: (Gym Space) The action space of the environment
+  :param n_env: (int) The number of environments to run
+  :param n_steps: (int) The number of steps to run for each environment
+  :param n_batch: (int) The number of batch to run (n_envs * n_steps)
+  :param reuse: (bool) If the policy is reusable or not
+  :param _kwargs: (dict) Extra keyword arguments for feature extraction
```

Comment:
Why `_kwargs` not `kwargs`?

Reply:
No strong feelings for `_kwargs` (copy-pasted from [stable-baselines](https://github.com/hill-a/stable-baselines/blob/63f208eb21c67fd1a50a691b98ee54cdabc1308a/stable_baselines/common/policies.py#L598)). Updated it to kwargs.